schema/helpers/mask.json

```
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Mask",
+    "description": "Mask for layer content.",
+    "allOf": [
+        {
+            "properties": {
+                "mode": {
+                    "title": "Mode",
+                    "$ref": "#/$defs/constants/mask-mode",
+                    "default": "a"
```

[lottie-docs/schema](https://lottiefiles.github.io/lottie-docs/schema/#/$defs/helpers/mask) says the default mask mode is `i`, are we ok changing it?

```suggestion
                    "default": "i"
```

---

Not sure where the `i` default is coming from, in AE the default mode for new masks is Add.

That said, BM always exports an explicit mode, and intersection by default sort of makes sense from a graphics API point or view - so keeping `i` sgtm.